tools/phrase/env.base

```diff
@@ -0,0 +1,2 @@
+PHRASE_ACCESS_KEY=
```

Comment:
Would we have to enter the keys before running it? This could easily lead to someone forgetting to remove them, why don't you gitignore this file? You'd also have to remove it from git now that it's already tracked.

Reply:
One has to create a copy of this file as `.env` and put the Phrase Project ID and Access Key in that file.

We don't track `.env` file on git, so I guess we won't have any problems.